etc/init.d/zfs-zed.in

```diff
@@ -45,8 +39,7 @@ ZED_PIDFILE="@runstatedir@/$ZED_NAME.pid"
 
 do_depend()
 {
-	before zfs-import
-	after sysfs
+	after sysfs zfs-mount
```

Comment:
Saying to run after zfs-mount should not be necessary. I do not think there is any need to enforce a strict ordering here. Leaving the order undefined would allow these scripts to run in parallel.